grails-app/init/io/xh/toolbox/BootStrap.groovy

```diff
@@ -49,6 +51,28 @@ class BootStrap {
             ]).save()
         }
     }
+    private void ensureRequiredConfigsCreated() {
+        Utils.configService.ensureRequiredConfigsCreated([
+                newsApiKey : [
+                        valueType   : 'string',
+                        defaultValue: '917b84920ec24d7198fd712e2e22b48a',
+                        groupName   : 'news',
+                        note        : 'Key used to access News API'
```

Comment:
Sorry to nitpick, but this is the kind of note/comment we can skip - it contains no new info relative to the name of the config itself.  It's OK not to have notes for these things if they are self-evident.